kai/reactive_codeplanner/task_manager/task_manager.py

```diff
@@ -183,6 +183,11 @@ def get_next_task(
         seed_tasks_only = max_depth == 0 and self.priority_queue.has_tasks_within_depth(
             0
         )
+        logger.info(
+            "seed tasks only: %s, priority_queue: %s",
+            seed_tasks_only,
+            self.priority_queue.task_stacks,
```

Comment:
What does this print?

Reply:
This was debugging when things were not doing what I wanted. 

I can remove sorry